.eslintrc.json

```diff
@@ -17,6 +17,9 @@
     ],
     "rules": {
         "@typescript-eslint/no-var-requires": 1,
-        "no-prototype-builtins": 0
+        "no-prototype-builtins": 0,
+        // TODO: We really should remove these and just fix the lint errors.
+        "@typescript-eslint/no-explicit-any": "warn",
+        "@typescript-eslint/no-unused-vars": "warn"
```

Comment:
We really should work to remove these, but there's well over 100 errors. We need to be more conscious about not using `any and should probably call it out on PRs.

Reply:
Following up on the above, a good number of the files in this PR are just lint fixes for unused imports, spaces, etc. So ~half the lint work has been done minus these two rules.